main.typ

```diff
@@ -19,7 +19,7 @@
 #outline(title: "List of Tables", target: figure.where(kind: table))
 #pagebreak()
 
-#include "changelog.typ"
+//#include "changelog.typ"
```

Comment:
Don't comment this out, just create a blank `changelog.csv` so your build works.